examples/wasm_ledger/test/test.ledger.js

```diff
@@ -225,8 +225,7 @@ describe("LEDGER TEST", function () {
     assert(signatureOk);
   });
 
-  // SKIP: because the ledger app is back to testnet2
-  it.skip("#transactionSignRawWithDevice() Testnet", async function() {
+  it("#transactionSignRawWithDevice() Testnet", async function() {
```

Comment:
Oh this one won't work. Since we moved away from the javascript lib for ledger we can't pass message like this. It need to be digest before. But I am fixing this.

Reply:
This will be in this PR https://github.com/Zondax/filecoin-rs/pull/157

Lets skip this for now.

Reply:
sounds good. Still, I did a bit of additional research on the `UnhandledPromiseRejectionWarning`. There is something bad going on there anyway.. but that seems to be an issue in ledger-rs